packages/rancher-vsphere/rancher-vsphere-cpi/package.yaml

```
@@ -1,4 +1,4 @@
 url: https://github.com/rancher/vsphere-charts.git
 subdirectory: charts/rancher-vsphere-cpi
-commit: 5bcff9e69b932cb9c86bd1061b2ee4b5367ecb60
-version: 102.0.0
+commit: 8b8e8cf13e9c971330bf96517fab1deec1f23b05
+version: 103.0.0
```

We should bump the minor version in the event of adding support for new k8s versions. 

```suggestion
version: 102.1.0
```